public/locales/en/common.json

```diff
@@ -1,12 +1,19 @@
 {
-  "title": "Beneath the Surface",
-  "placeholder": "Sample text change here. Vestibulum finibus ante vehicula, maximus erat vel, mollis ex. In erat elit, maximus at enim nec, tincidunt vulputate est. Quisque rhoncus egestas egestas. Vivamus ut iaculis ante, id volutpat lacus. Proin sit amet rutrum arcu. Cras elementum ligula nec volutpat varius. Maecenas viverra lectus quis velit finibus tempor. Donec et ligula enim. Sed eget scelerisque enim. Duis quis metus ac nisi accumsan eleifend id quis massa.",
-  "header": {
-    "brandAlt": "Crane logo with the text: Beneath the Surface",
-    "topics": "Topics",
-    "downloads": "Downloads",
-    "purchase": "Purchase the Book",
-    "about": "About Us",
-    "blurb": "We desire to see a prayer movement for Japan's spiritual breakthrough"
-  }
+    "title": "Beneath the Surface",
+    "placeholder": "Sample text change here. Vestibulum finibus ante vehicula, maximus erat vel, mollis ex. In erat elit, maximus at enim nec, tincidunt vulputate est. Quisque rhoncus egestas egestas. Vivamus ut iaculis ante, id volutpat lacus. Proin sit amet rutrum arcu. Cras elementum ligula nec volutpat varius. Maecenas viverra lectus quis velit finibus tempor. Donec et ligula enim. Sed eget scelerisque enim. Duis quis metus ac nisi accumsan eleifend id quis massa.",
+    "header": {
+        "brandAlt": "Crane logo with the text: Beneath the Surface",
+        "topics": "Topics",
+        "downloads": "Downloads",
+        "purchase": "Purchase the Book",
+        "about": "About Us",
+        "blurb": "We desire to see a prayer movement for Japan's spiritual breakthrough"
+    },
+    "home": {
```

Comment:
Home stuff feels like we should put it in it's own JSON? Feel free to disagree though

Reply:
yeah i think at some point we should tidy up all our jsons, potentially we wont even end up with a common since it won't make a lot of sense to someone new looking at it